config/initializers/session_store.rb

```diff
@@ -21,7 +21,7 @@
                 when "memory" then  :memory_store
                 when "cache" then   :dalli_store
                 else
-                  raise "session_store, '#{evm_store}', invalid. Should be one of 'sql', 'memory', 'cache'"
+                  raise "session_store, '#{evm_store}', invalid. Should be one of 'sql', 'memory', 'cache'.  Source configuration: #{config_file}"
```

Comment:
As soon as I saw what was in my vmdb.yml.db, I found the problem in 1 minute by grepping for the contents of the file.